sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java

```diff
@@ -13,9 +13,9 @@
  * Additional parameters for a set of operations, such as: Files_create,
  * Files_setHTTPHeaders.
  */
-@JacksonXmlRootElement(localName = "file-HTTP-headers")
+@JacksonXmlRootElement(localName = "file-http-headers")
```

Comment:
Is this an actual XML element that is expected or is this an artifact of AutoRest codegen?